src/components/dropdown-add-btn/DropdownButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, ReactElement } from 'react'
+import { SxProps } from '@mui/material'
+
+import AppButton from '~/components/app-button/AppButton'
+
+import { ButtonVariantEnum, SizeEnum } from '~/types'
+import { styles } from '~/components/dropdown-add-btn/DropdownButton.styles'
+
+interface DropdownButtonProps {
+  handleOnClick: () => void
+  icon: ReactElement
+  sx?: SxProps
+  value: string
+}
+
+const DropdownButton: FC<DropdownButtonProps> = ({
+  handleOnClick,
+  icon,
+  sx,
+  value
+}) => {
+  return (
+    <AppButton
+      disableRipple
+      fullWidth
+      onClick={handleOnClick}
+      size={SizeEnum.Medium}
+      sx={{ ...styles.optionsButton, ...sx }}
```

Comment:
use spliceSx function to combine both styles

Reply:
done)